sql-plugin/src/main/scala/org/apache/spark/sql/rapids/AggregateFunctions.scala

```diff
@@ -529,3 +529,42 @@ abstract class GpuLastBase(child: Expression)
   override lazy val deterministic: Boolean = false
   override def toString: String = s"gpulast($child)${if (ignoreNulls) " ignore nulls"}"
 }
+
+/**
+ * Collects and returns a list of non-unique elements.
+ *
+ * FIXME Not sure whether GPU version requires the two offset parameters. Keep it here first.
```

Comment:
What are the two offset parameters?
The signature of the GPU `collect()` window function currently takes only a child expression (column).

Reply:
Yes, the cuDF `collect()` takes no parameters other than child expression.
This signature is copied from CPU version here .https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/collect.scala#L102
```scala
case class CollectList(
    child: Expression,
    mutableAggBufferOffset: Int = 0,
    inputAggBufferOffset: Int = 0) extends Collect[mutable.ArrayBuffer[Any]] {
```
I am fine to remove them if not required by GPU version.

Reply:
Thanks for the clarification, @firestarman. Let us keep the signatures identical for now, lest they be flagged during an audit for interface compatibility. It might be alright not to use them for the moment.